src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  flex-grow: 0;
+  width: 200px;
+  padding: 16px;
+  margin-right: 48px;
+  margin-bottom: 46px;
+
+  background: $white;
+  border: 1px solid $grayElements;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: auto;
+    margin-top: 16px;
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    padding-bottom: 4px;
+
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+    color: $mainAccent;
+  }
+
+  &__product-code {
+    padding-bottom: 16px;
+
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    font-size: 10px;
+    color: #616070;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+
+    padding-bottom: 16px;
+  }
+
+  &__price-text {
+    display: flex;
+    align-items: flex-end;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+  }
+
+  &__price-value {
+    display: flex;
+    align-items: flex-end;
+    text-align: right;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $mainAccent;
+  }
+
+  &__buy-button {
+    font-family: "Roboto", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 14px;
+
```

Comment:
You already used the same font in src/styles/normalize.scss
`html {
  font-family: "Roboto", sans-serif;
}`
Do we need it there?